packages/reputation-miner/bin/index.js

```diff
@@ -50,14 +50,16 @@ class RetryProvider extends ethers.providers.JsonRpcProvider {
   }
 
   getNetwork(){
-    return backoff(() => super.getNetwork(), {retry: this.attemptCheck});
+    const that = this;
```

Comment:
Why?

Reply:
I saw an error around `this.attemptCheck` and correctly using `.bind` is a big blind spot for me, so reverted to the `that=this` pattern out of safety. I think the actual issue here though is that I declared the function as static for some reason (even though it uses `this`), and so won't be available on the instances, just on the class itself.

Reply:
JS scoping is not my strong suit either so I'm going to assume you did an excellent job.